7.4/fpm/ubuntu/Dockerfile

```diff
@@ -1,6 +1,15 @@
 FROM ubuntu:18.04
 
+ONBUILD ARG XDEBUG_ENABLED="0"
+ONBUILD RUN test "$XDEBUG_ENABLED" = "1" && /build/install_xdebug.sh || echo "Skipping Xdebug"
+
+ENV PHP_INI_DIR /etc/php/7.4/fpm
+
 COPY build /build
+COPY docker-php-ext-enable /usr/local/bin/
+
+RUN chmod +x /build/install_xdebug.sh
+RUN chmod +x /usr/local/bin/docker-php-ext-enable
```

Comment:
This can be omitted if you `chmod +x` those files directly in the repository. Docker should copy also the permissions for those files.

Reply:
Yep, good idea. Done.